Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM golang:1.5
+
+ENV PROJECT github.com/mozilla-services/go-bouncer
+
+COPY . /go/src/$PROJECT
+
+ENV GOPATH /go/src/$PROJECT/Godeps/_workspace:$GOPATH
```

Comment:
Put this in line 4. Each instruction in a Dockerfile creates a new layer. Moving it to line 4 will be more cache friendly.